probe/ebpf/flow.h

```diff
@@ -82,17 +110,29 @@ struct flow_metrics {
 	__u64 ba_bytes;
 };
 
+struct l2 {
+	__u64                  key;
+	__u64                  layers_path;
+
+	__u32                  next_layer_offset;
+	struct link_layer      link_layer;
+	__u64                  last;
+};
+
 struct flow {
 	__u64                  key;
+	__u64                  key_1st;
 
 	// define layers set in the flow
-	__u8                   layers;
+	__u64                  layers_path;
 
 	// ifIndex of the interface seing the packet
 	__u32                  ifindex;
 
 	// layers information
+	__u8                   layers_info;
 	struct link_layer      link_layer;
+	struct network_layer   network_layer_1st;
```

Comment:
Wondering if `_outer` would be clearer that `_1st`

Reply:
yeah the idea of 1st, it's the 1st layer on the path, but outer may be _cleaner_

Reply:
CleaNer, I don't know. But CleaRer is important :-)